src/test/java/com/google/cloud/tools/maven/it/RunAsyncMojoIntegrationTest.java

```diff
@@ -63,6 +66,20 @@ private void test(String name, SupportedDevServerVersion version)
       String urlContent = UrlUtils.getUrlContentWithRetries(getServerUrl(), 60000, 1000);
       assertThat(urlContent, containsString("Hello from the App Engine Standard project."));
       assertThat(urlContent, containsString("TEST_VAR=testVariableValue"));
+
+      File expectedLog =
+          Paths.get(
+                  "target",
```

Comment:
Is this explicit pathing the only way to get to this file?

Reply:
Not sure, is there a way to get the build directory from within the integration tests @loosebazooka?